src/RuleCustomisationPolicyAwareConfigInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer;
```

Comment:
```suggestion
namespace PhpCsFixer\Config;
```

time to stop bloating main namespace

Reply:
> time to stop bloating main namespace

I totally agree